### DISCUSSION TITLE
Error getting image files to display correctly

### DISCUSSION THREAD

**Discussion by justice-dazzle** [requarks/wiki#3593] [Error / Bug Report]
I am getting an error when displaying image files. 

1. Upload files (tried .png and .jpeg)
2. Select newly uploaded files and insert into page.
3. Dialog acts like the image inserted successfully, but it does not appear in the editor. 
4. Looking at the command line log running on the server, we get this error: "warn: Cannot read property 'data' of undefined"

I have rebooted and installed the latest server OS, same issue.

Running On Windows Server 2019, Wiki.js version: 2.2.51, Node.js version: 12.14.1, MySQL version: 8.0.19 

**Describe the bug**

**To Reproduce**

**Expected behavior**

**Screenshots**

**Host Info (please complete the following information):**
 - OS: [e.g. Ubuntu 18.04, Docker]
 - Wiki.js version: [e.g. 2.0.1]
 - Database engine: [e.g. postgres 9.7, mysql 5.7]

**Additional context**

**Comment by Smankusors** [comment]
Hmm Wiki.js 2.2.... Can you try with Wiki.js 2.4.x?
